lib/extensions/string_extension.rb

```diff
@@ -0,0 +1,15 @@
+module StringExtension
+  def true?
+    /(true|t|yes|y|1)$/i.match(to_s).present?
+  end
+
+  def false?
+    to_s.strip.empty? || /(false|f|no|n|0)$/i.match(to_s).present?
+  end
+
+  def to_bool
```

Comment:
This isn't being used anywhere.  Do you need it at this point?

Reply:
Hmm, not really. I guess I can just leave the true? / false? methods. I will remove it and update the tests as at the moment to_bool is the method being tested than implicitly uses the others.